api/scripts/extract_sentry_user_emails.py

```diff
@@ -0,0 +1,41 @@
+import requests
+
+
+def generate_email_list(sentry_api_token: str, sentry_org_slug: str, sentry_project_id: str):
```

Comment:
is `sentry_project_id` our organization id?

Reply:
Nope, there's and org id and project id. An organization can have multiple projects.

Reply:
Nope, it's and id that identifies a unique project under and org. I think orgs also have ids but I just used the slug because it's easier to remember.